_source/_plugins/speakerdeck_tag.rb

```diff
@@ -0,0 +1,26 @@
+module Jekyll
+    # A simple tag for Speakerdeck include using official script
+    class SpeakerdeckTag < Liquid::Tag
+      @@width = 640
+      @@height = 510
+
+      def initialize(name, id, tokens)
+        super
+        @id = id
+      end
+
+      def render(context)
+        @id_stripped = @id.strip.tr('"', '')
+        %(<p>
```

Comment:
I would suggest changing this to be a div:

```
<div style="margin: 0 auto 1.25rem auto; max-width: 700px">
```

We only have one post with Speaker Deck slides embedded, but this is what I used:

```
<div style="margin: 0 auto 1.25rem auto; max-width: 700px">
<script async class="speakerdeck-embed" data-id="2eba574e453e4b1684a9f1509564c798" data-ratio="1.77777777777778" src="//speakerdeck.com/assets/embed.js"></script>
</div>
```

The post is at https://developer.okta.com/blog/2020/04/27/mobile-development-ionic-react-native-jhipster, and it looks like I need to fix our CSP for speakerdeck.com.

```
Refused to load the script 'https://speakerdeck.com/assets/embed.js' because it violates the following Content Security Policy directive:
```